src/NavLights.cpp

```diff
@@ -200,8 +200,8 @@ void NavLights::Render(Graphics::Renderer *renderer)
 
 	const bool bVBValid = m_billboardVB.Valid();
 	const bool bHasVerts = !m_billboardTris.IsEmpty();
-	const bool bVertCountEqual = bVBValid && (m_billboardVB->GetVertexCount() == m_billboardTris.GetNumVerts());
-	if( bHasVerts && (!bVBValid || !bVertCountEqual) )
```

Comment:
Random note, but at some point we're going to have to be consistent in our local variable naming schemes. I'm guessing the b is for boolean though :)

Reply:
yes, really I should have just renamed them to something like `isValid`, but I have a bad habit of just going along with whatever the local (file/function local) scheme already happens to be instead of enforcing ours!